Use native ReflectionClass

janbarasek opened the pull request:
Lowest dependency.

janbarasek committed:
Fix coding standard by Nette 3 specification.

janbarasek committed:
Remove redundant composer.lock file.!

janbarasek committed:
README: Better package name.

janbarasek committed:
Move classes to src

janbarasek committed:
Add support for PHP 7.1+ and define automated tests for PhpStan and codestyle.

janbarasek committed:
NotOrmTracyPanel: Fix coding standard

janbarasek committed:
Fix PhpStan errors

janbarasek committed:
Back support for getNotOrm() and getPdo().

janbarasek committed:
Fix PhpStan

janbarasek committed:
Fix PhpStan

janbarasek committed:
Merge branch 'master' of github.com:Zemistr/notorm-tracy-panel

janbarasek committed:
Use native reflectionClass.

janbarasek commented:
Please use a squash strategy to merge my broken commits.

Zemistr commented:
@janbarasek Sorry for the late reply. I was busy, and I wasn't checking GitHub for kinda long. :| 

Can you rebase your PRs to master instead of the merge?

janbarasek commented:
@Zemistr I do not know how. When merging, select the "squash" button, it will happen automatically.

dependabot-preview[bot] committed:
Upgrade to GitHub-native Dependabot

janbarasek committed:
Merge pull request #2 from baraja-core/dependabot/add-v2-config-file

Upgrade to GitHub-native Dependabot

dependabot[bot] committed:
Update symplify/easy-coding-standard requirement from ^7.2 to ^9.4 (#5)

Updates the requirements on [symplify/easy-coding-standard](https://github.com/symplify/easy-coding-standard) to permit the latest version.
- [Release notes](https://github.com/symplify/easy-coding-standard/releases)
- [Commits](https://github.com/symplify/easy-coding-standard/compare/v7.2.0...9.4.1)

---
updated-dependencies:
- dependency-name: symplify/easy-coding-standard
  dependency-type: direct:development
...

Signed-off-by: dependabot[bot] <support@github.com>

Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

dependabot[bot] committed:
Update symplify/easy-coding-standard requirement from ^9.4 to ^10.0 (#6)

Updates the requirements on [symplify/easy-coding-standard](https://github.com/symplify/easy-coding-standard) to permit the latest version.
- [Release notes](https://github.com/symplify/easy-coding-standard/releases)
- [Commits](https://github.com/symplify/easy-coding-standard/compare/9.4.1...10.0.0)

---
updated-dependencies:
- dependency-name: symplify/easy-coding-standard
  dependency-type: direct:development
...

Signed-off-by: dependabot[bot] <support@github.com>

Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>
